rtt/base/TaskCore.cpp

```diff
@@ -53,6 +55,7 @@ namespace RTT {
            ,mTaskState(initial_state)
            ,mInitialState(initial_state)
            ,mTargetState(initial_state)
+           ,cName(strdup(""))
```

Comment:
Add a `TaskCore` constructor which directly takes a `const string &` argument, analog to [TaskContext](https://github.com/orocos-toolchain/rtt/pull/187/files#diff-a739de4d537602c73893d46c20da0a31R70)?